### ISSUE TITLE
[Feature Request]: Please bring back the "Switch to GSdx Software rendering when an FMV plays" hack

### ISSUE THREAD

**Issue by osuperjames** [PCSX2/pcsx2#9619]
### Description

Hi, friends.

I'm playing Dragon Quest VIII - Journey of the Cursed King on PCSX2 Nightly v1.7.4788 and when I call the Great Sabrecat the FMV animation shows a black bar at the bottom of the screen, like the screenshot [here.](https://i.imgur.com/LSW5irx.jpg)

Searching online I got the resolution: turn on the software mode, but I saw that in the past the PCSX2 used to have a option to do it automatically when a FMV pops up. I don't know why you remove this, but I'm coming here to ask to bring this option again.

See, I play on my PC, but there is people who play on Steam Deck without a keyboard or want to lay down on couch and play without any need to press f9 constantly, so I think that justifies the reimplementation of this option.

Thanks for your work!

### Reason

I already said in the previous message.

### Examples

[Here](https://i.imgur.com/JvX1UX3.jpg) is a screenshot from PCSX2 1.6.0 Stable version.

**Comment by stenzek** [comment]
It's literally there under game fixes.

**Comment by osuperjames** [comment]
> It's literally there under game fixes.

I'm sorry, but I spent a half hour looking for "game fixes" on nightly version, I only found on stable version. Where is located "Switch to GSdx Software rendering when an FMV plays" or "Game fixes" exactly?

Sorry for the dumb question.

**Comment by RedDevilus** [comment]
Per-game and Advanced Settings is the crutch of the problem. But yeah this is a support request as it was never gone.

**Comment by osuperjames** [comment]
> Per-game and Advanced Settings is the crutch of the problem. But yeah this is a support request as it was never gone.

Thanks!

To help anyone facing the same issue I'll leave here how I managed to solve this, step by step.

1. Check "Show Advanced Settings" on "Tools" menu [that's why I didn't found "Game Fixes" when I looked before];
2. Go to your title on your game list and right click on it, select "Properties" [this will make the change only for the selected game];
3. Navigate to "Game Fixes" and check "Use Software Renderer For FMVs".
4. Enjoy.

Thanks, guys! You are the best.
